GraphQL: add cache clearing instruction (#661)
api-platform/docs: core/graphql.md

    $ composer require webonyx/graphql-php

> You can now use GraphQL at the endpoint: `http://localhost/graphql`. In some case, you might have to clear the cache of your application.

Reviewer: ```suggestion
You can now use GraphQL at the endpoint: `http://localhost/graphql`. Then, clear the cache of your application by running `bin/console cache:clear`.
```
